### ISSUE TITLE
No bookmark star icon

### ISSUE THREAD

**Issue by neuberoliveira** [everettjf/XBookmark#9]
Hi, my only problem with the plugin is the fact that the star icon on the editor side bar not appear

![captura de tela 2016-07-26 as 21 41 43](https://cloud.githubusercontent.com/assets/3378304/17160215/d4017424-5379-11e6-872e-09d82ea0dcd1.png)

Xcode: Version 7.3.1 (7D1014)
OSX: El Capitan 10.11.5 (15F34)

**Comment by everettjf** [comment]
This plugin has an known bug that has not fixed:
If you open more than one project(or workspace) , the bookmark icon only works for the first one.
1. Please try quit Xcode and open only one project (workspace).
2. Or try download the source zip , unzip it, and run.

**Comment by gdahilig** [comment]
I just tried to install it with the latest Xcode and I also don't get any Star bookmark icons.  

Xcode: v.8.3.2 (8E2002)
OS: macOS Sierra v.10.12.5 (16E195)

**Comment by Tough-Guy** [comment]
If I'm not wrong, it is only working on the projects that are created after the installation of the plugin and only one active project at a time.

**Comment by everettjf** [comment]
@Tough-Guy Yes, It only works for the first opened project after Xcode start. (not the active proejct ) . But it is enough for me ....so I did not fix it ...Sorry.

And now ,it not support Xcode9. Maybe support in the future ( a few months)...
